server/tests/belga_360_archive_test.py

```diff
@@ -0,0 +1,109 @@
+    def setUp(self):
+        self.provider = Belga360ArchiveSearchProvider(dict())
+        self.query = {
+            'size': 50,
+            'from': 50,
+            'query': {
+                'filtered': {
+                    'query': {
+                        'query_string': {
+                            'query': 'test query'
+                        },
+                    },
+                },
+            },
+        }
+
+    def test_instance(self):
+        self.assertEqual('Belga 360 Archive', self.provider.label)
+        self.assertIsInstance(self.provider, superdesk.SearchProvider)
+
+    def test_find_params(self):
+        self.provider.session.get = MagicMock()
+        self.provider.find(self.query)
+        url = self.provider.base_url + 'archivenewsobjects'
+        params = {'start': 50, 'pageSize': 50, 'searchText': 'test query'}
+        self.provider.session.get.assert_called_with(url, params=params)
+
+    def test_format_list_item(self):
+        item = self.provider.format_list_item(get_belga360_item())
+        guid = 'urn:belga.be:360archive:39670442'
+        assert item['type'] == 'text'
+        assert item['mimetype'] == 'application/vnd.belga.360archive'
+        assert item['_id'] == guid
+        assert item['guid'] == guid
+        assert item['extra']['bcoverage'] == guid
+        assert item['headline'] == 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+        assert item['name'] == ''
+        assert item['description_text'] == ''
+        assert item['creditline'] == 'BELGA'
+        assert item['source'] == 'BELGA'
+        assert item['language'] == 'fr'
+        assert item['abstract'] == (
+            'Vivamus rutrum sapien a purus posuere eleifend. Integer non feugiat sapien. Proin'
+            ' finibus diam in urna vehicula accumsan'
+        )
+        assert item['body_html'] == (
+            'Morbi lacus ex, molestie id ullamcorper quis, scelerisque quis lectus.\n'
+            ' Phasellus laoreet turpis nunc, vitae porttitor sapien ultricies non.\n'
+            ' Nullam fringilla justo vitae ex commodo vulputate.\n In bibendum diam vitae condimentum scelerisque.\n'
+            ' Integer dapibus turpis augue, a varius diam ornare in.\n Donec aliquam cursus posuere.'
+        )
+
+    def test_find_item(self):
+        with HTTMock(archive_mock):
+            items = self.provider.find(self.query)
+        assert len(items.docs) == 2
+        assert items._count == 25000
+
+    def test_fetch(self):
+        response = DetailResponse()
+        response.json = MagicMock(return_value=get_belga360_item())
+        self.provider.session.get = MagicMock(return_value=response)
+
+        item = self.provider.fetch('urn:belga.be:360archive:39670442')
+
+        url = requests.Request(
+            'GET', self.provider.base_url + 'archivenewsobjects/39670442'
+        ).prepare().url
+        self.provider.session.get.assert_called_with(url)
```

Comment:
I think this is a correct assert for fetch by guid: `self.provider.session.get.assert_called_with(url, params={})`